### PR TITLE
Task to drf Class-Based Views added

### PR DESCRIPTION
**Pull request by Y-Havryliv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,17 @@
...
+- the `AuthorViewSet` viewset in `views.py`.
+
+The `Author` model has the following fields:
+- first_name (with the `max_length` of 64);
+- last_name (with the `max_length` of 64);
+- pseudonym (with the `max_length` of 64, can be null);
+- age (integer field);
+- retired (boolean field).
+
+**Please note:** you should also modify `author.urls.py` to make things work.
```
Comment (danylott) [inline README.md:17]:
```suggestion
**Please note:** you should also modify `author/urls.py` to make things work.
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,17 @@
-# Python Boilerplate For GitHub Tasks
+# ModelViewSet Implementation
```
Comment (danylott) [inline README.md:1]: Author CRUD Implementation

File: README.md [hunk 3]
```diff
@@ -1,4 +1,17 @@
...
 
 **Please note:** read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md)
-before starting.
+before start.
+
+In this task, you should implement:
```
Comment (danylott) [inline README.md:6]: - full CRUD API for Author

File: author/tests/tests.py [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+from rest_framework import status
+
+from rest_framework.test import APIClient
+from rest_framework.viewsets import ModelViewSet
+
+from author.models import Author
+from author.serializers import AuthorSerializer
+from author.views import AuthorViewSet
+
+AUTHORS_URL = "/api/authors/"
```
Comment (danylott) [inline author/tests/tests.py:11]: Use `reverse` here. It's a best practice

**Review by danylott — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1,4 +1,15 @@
-# Python Boilerplate For GitHub Tasks
+# Author CRUD Implementation
 
 **Please note:** read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md)
-before starting.
+before start.
```
Comment (alenatovstukha) [inline README.md:4]: starting.

File: README.md [hunk 6]
```diff
@@ -1,4 +1,15 @@
...
+
+In this task, you should implement full CRUD API for `Author`.
```
Comment (alenatovstukha) [inline README.md:6]: the `Author` model.

File: README.md [hunk 7]
```diff
@@ -1,4 +1,15 @@
...
+
+The `Author` model has the following fields:
+- first_name (with the `max_length` of 64);
```
Comment (alenatovstukha) [inline README.md:9]: название всех полей выделить `кодом`

File: README.md [hunk 8]
```diff
@@ -1,4 +1,15 @@
...
+- last_name (with the `max_length` of 64);
+- pseudonym (with the `max_length` of 64, can be null);
+- age (integer field);
+- retired (boolean field).
+
+**Please note:** you should also modify `author/urls.py` to make things work.
```
Comment (alenatovstukha) [inline README.md:15]: the `author/urls.py` file

**Pull request merged**